packages/block-library/src/site-logo/edit.js

```diff
@@ -338,6 +344,7 @@ export default function LogoEdit( {
 				onSelect={ onSelectLogo }
 				onError={ onUploadError }
 			/>
+			<ToolbarButton onClick={ onRemoveLogo } icon={ trash } />
```

Comment:
I like the use of the trash icon, but should we also add a `label` prop so that we can display a tooltip if the user hovers over the icon?

![image](https://user-images.githubusercontent.com/14988353/133707339-8cb61441-7884-4914-93df-a9d18655f1a1.png)

Reply:
Absolutely -- done!